src/mds/MDSRank.cc

```diff
@@ -3781,6 +3784,29 @@ void MDSRankDispatcher::handle_conf_change(const ConfigProxy& conf, const std::s
   if (changed.count("mds_enable_op_tracker")) {
     op_tracker.set_tracking(conf->mds_enable_op_tracker);
   }
+  if (changed.count("mds_extraordinary_events_dump_interval")) {
+    reset_event_flags();
+    extraordinary_events_dump_interval =
+      conf.get_val<std::chrono::seconds>("mds_extraordinary_events_dump_interval").count();
+
+    std::string debug_mds = g_conf().get_val<std::string>("debug_mds");
+    std::stringstream ss(debug_mds.substr(debug_mds.find("/") + 1));
+    ss >> inmemory_log_level;
+
+    //Enable the logging only with lower level debugging
+    if (extraordinary_events_dump_interval) {
+      if (inmemory_log_level < 15) {
```

Comment:
I checked this more, it should be `inmemory_log_level < 10`. As default if the log level is larger than or equal to `10` that logs should have already outputed.